lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -99,30 +101,37 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
     @Input()
     displayTags = false;
```

Comment:
You may need similar input for categories. Notice that metadata component is used in couple places, not only in viewer which we change but also for example in some places in outside content-services like process-services-cloud and we don't want probably affect other places

Reply:
Added